src/main/java/kitchenpos/menu/ui/MenuRestController.java

```diff
@@ -0,0 +1,39 @@
+package kitchenpos.menu.ui;
+
+import java.net.URI;
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import kitchenpos.menu.application.MenuService;
+import kitchenpos.menu.dto.MenuRequest;
+import kitchenpos.menu.dto.MenuResponse;
+
+@RestController
+public class MenuRestController {
+	private final MenuService menuService;
+
+	public MenuRestController(final MenuService menuService) {
+		this.menuService = menuService;
+	}
+
+	@PostMapping("/api/menus")
+	public ResponseEntity<MenuResponse> create(@RequestBody final MenuRequest menuRequest) {
```

Comment:
```suggestion
	public ResponseEntity<MenuResponse> create(@RequestBody @Valid final MenuRequest menuRequest) {
```
`@Valid`를 이용하여 `MenuRequest`에 `validation`을 추가해 볼 수 도 있을 것 같습니다 😄 
그럼 `Request` 안에 있는 값들에 대해서 별도의 추가 확인을 안해도 됩니다 😄 
이 부분도 참고해주시면 감사하겠습니다 🙇 

도움되실 만한 내용 공유 드립니다 🙇 
> https://meetup.toast.com/posts/223